inc/html.class.php

```diff
@@ -6121,7 +6121,7 @@ static function showCheckboxMatrix(array $columns, array $rows, array $options =
     *
     * @since 9.1
    **/
-   static function ajaxForm($selector, $success = "console.log(html);") {
+   static function ajaxForm($selector, $success = "console.log(html);", $error = "console.log(html)", $complete = '') {
```

Comment:
Should it be logged using console.error ?
```suggestion
   static function ajaxForm($selector, $success = "console.log(html);", $error = "console.error(html)", $complete = '') {
```

Reply:
This is indeed a good idea.